scst-store/connect-to-database.hbs.md

```diff
@@ -0,0 +1,43 @@
+# Connecting to the Postgres Database
+
+To connected to the Postgres Database, you will need the following values:
+* database name
+* username
+* password
+* database host
+* database port
+
+1. To obtain the database name, username, and password, run the following commands:
+    ```shell
+    db_name=$(kubectl get secret postgres-db-secret -n metadata-store -o json | jq -r '.data.POSTGRES_DB' | base64 -d)
+    db_username=$(kubectl get secret postgres-db-secret -n metadata-store -o json | jq -r '.data.POSTGRES_USER' | base64 -d)
+    db_password=$(kubectl get secret postgres-db-secret -n metadata-store -o json | jq -r '.data.POSTGRES_PASSWORD' | base64 -d)
+    ```
+    If the password was auto-generated, the above `password` command will return an empty string. Instead, run the following command:
+    ```shell
+    db_password=$(kubectl get secret postgres-db-password -n metadata-store -o json | jq -r '.data.DB_PASSWORD' | base64 -d)
+    ```
+1. In a separate terminal, run the following command:
+
+    ```shell
+    kubectl port-forward service/metadata-store-db 5432:5432 -n metadata-store
+    ```
+    
+    With this, database host and port values will be the following:
```

Comment:
"Set the database host and port values on the first terminal with the following"